api/system_can.dbc

```diff
@@ -0,0 +1,317 @@
+VERSION ""
```

Comment:
I think we should put this in .gitignore, so if people start uploading different DBC files we will not keep changing this. Probably best not to push this.